otelcollector/scripts/setup.sh

```diff
@@ -35,24 +35,24 @@ gem install deep_merge
 gem install re2
 
 echo "Installing mdsd..."
-if [ "${ARCH}" != "amd64" ]; then
-  wget https://github.com/Azure/prometheus-collector/releases/download/azure-mdsd-1.23.3/azure-mdsd_1.23.4-build.master.28_aarch64.rpm
-  sudo tdnf install -y azure-mdsd_1.23.4-build.master.28_aarch64.rpm
-else
-  wget https://github.com/Azure/prometheus-collector/releases/download/azure-mdsd-1.23.3/azure-mdsd_1.23.4-build.master.28_x86_64.rpm
-  sudo tdnf install -y azure-mdsd_1.23.4-build.master.28_x86_64.rpm
-fi
+# if [ "${ARCH}" != "amd64" ]; then
+#   wget https://github.com/Azure/prometheus-collector/releases/download/azure-mdsd-1.23.3/azure-mdsd_1.23.4-build.master.28_aarch64.rpm
+#   sudo tdnf install -y azure-mdsd_1.23.4-build.master.28_aarch64.rpm
+# else
+#   wget https://github.com/Azure/prometheus-collector/releases/download/azure-mdsd-1.23.3/azure-mdsd_1.23.4-build.master.28_x86_64.rpm
+#   sudo tdnf install -y azure-mdsd_1.23.4-build.master.28_x86_64.rpm
+# fi
 
 # Install this way once moving to the Mariner published RPMs:
-# sudo tdnf install -y azure-mdsd
+sudo tdnf install -y azure-mdsd
```

Comment:
We should pin this to a version so that we don't take a new release without knowing it

Reply:
Thanks Grace! I updated this to fetch a specific version.